arrows/vxl/high_pass_filter.cxx

```diff
@@ -188,9 +188,9 @@ ::horizontal_box_bidirectional_pass( vil_image_view< PixType > const& grey,
   PixType diff1 = 0;
   PixType diff2 = 0;
 
-  for( decltype( +nj ) j{ 0 }; j < nj; ++j )
+  for( unsigned j = 0; j < nj; ++j )
```

Comment:
...as long as `nj` is also `unsigned`, I am okay with this; I mainly don't want to have `size_t` vs. `unsigned` mismatches.